src/repositories/friendship.repository.ts

```diff
@@ -66,4 +70,23 @@ export class FriendshipRepository extends Repository {
     });
     return body.friendship_status;
   }
+
+  async addAndRemoveBesties(addList?: string[] | number[], removeList?: string[] | number[]) {
```

Comment:
This should be named `setBesties` and should probably take an object like this: `{add?: Array<string | number>, remove?: Array<string | number>} = {}`.